### PR TITLE
Pushing updated strings for localization

### PR DESCRIPTION
**Pull request by adityamulik**
Hello,
This PR is for the recent translated strings retrieved from Memsource and the strings were uploaded to Memsource from the master branch on 10th June, 2022.
Commit belongs to Sprint 12 (https://issues.redhat.com/browse/AAP-4533)

### CODE REVIEW

File: locale/es.po [hunk 1]
```diff
@@ -4,27 +4,26 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: @lingui/cli\n"
-"Language: es\n"
+"Language: en\n"
```
Comment (himdel) [inline locale/es.po:7]: these should be specifying the actual language .. `en` only belongs in the english version
(this is happening in all the files)

File: locale/es.po [hunk 2]
```diff
@@ -701,11 +698,11 @@ msgstr "Crear un nuevo espacio de nombres"
 
 #: src/components/user-form/user-form.tsx:97
 #~ msgid "Create a password using at least 9 characters, including special characters , ex <!@$%>. Avoid using common names or expressions."
-#~ msgstr "Cree una contraseña con, al menos, 9 caracteres, incluidos los caracteres especiales, por ejemplo, <!@$%>. Evite utilizar nombres o expresiones comunes."
+#~ msgstr "Create a password using at least 9 characters, including special characters , ex <!@$%>. Avoid using common names or expressions."
 
 #: src/components/user-form/user-form.tsx:99
 msgid "Create a password using at least {minLength} characters, including special characters , ex <!@$%>. Avoid using common names or expressions."
-msgstr ""
+msgstr "Cree una contraseña con, al menos, 9 caracteres, incluidos los caracteres especiales, por ejemplo, {minLength}. Evite utilizar nombres o expresiones comunes."
```
Comment (himdel) [inline locale/es.po:705]: This is wrong, the previous version ^ just needs to be adjusted for `9` -> `{minLength}`. The new version seems to treat the count as an example.

```suggestion
msgstr "Cree una contraseña con, al menos, {minLength} caracteres, incluidos los caracteres especiales, por ejemplo, <!@$%>. Evite utilizar nombres o expresiones comunes."
```

File: locale/es.po [hunk 3]
```diff
@@ -701,11 +698,11 @@ msgstr "Crear un nuevo espacio de nombres"
 
 #: src/components/user-form/user-form.tsx:97
 #~ msgid "Create a password using at least 9 characters, including special characters , ex <!@$%>. Avoid using common names or expressions."
-#~ msgstr "Cree una contraseña con, al menos, 9 caracteres, incluidos los caracteres especiales, por ejemplo, <!@$%>. Evite utilizar nombres o expresiones comunes."
+#~ msgstr "Create a password using at least 9 characters, including special characters , ex <!@$%>. Avoid using common names or expressions."
```
Comment (himdel) [inline locale/es.po:701]: Any reason the commented out obsolete translations get rewritten to English? These could still be useful when merging into older releases.

(That said, it's not *really* a problem, if it's nontrivial to fix, feel free to ignore.)

File: locale/fr.po [hunk 4]
```diff
@@ -50,7 +49,7 @@ msgstr "<0/> a été retiré de <1/>"
 
 #: src/components/collection-detail/collection-info.tsx:74
 msgid "<0>Note:</0> Installing collections with ansible-galaxy is only supported in ansible 2.9+"
-msgstr "<0>Remarque:</0> L'installation de collections avec ansible-galaxy n'est prise en charge que dans ansible 2.9+"
+msgstr "<0>Remarque : L'installation de collections avec ansible-galaxy n'est prise en charge que dans ansible 2.9+"
```
Comment (himdel) [inline locale/fr.po:52]: The translators seem to have removed the closing tag here, that won't work, we need to keep these balanced and matching the original.
```suggestion
msgstr "<0>Remarque:</0> L'installation de collections avec ansible-galaxy n'est prise en charge que dans ansible 2.9+"
```

File: locale/fr.po [hunk 5]
```diff
@@ -66,13 +65,13 @@ msgstr "<0>Note:</0> les liens suivants sont sans doute bloqués par votre navig
 
 #: src/containers/namespace-detail/namespace-detail.tsx:336
 msgid "<0>Note:</0> Use this URL to configure ansible-galaxy to upload collections to this namespace. More information on ansible-galaxy configurations can be found <1>here</1><2> </2><3/>."
-msgstr "<0>Remarque:</0> Utilisez cette URL pour configurer ansible-galaxy afin de télécharger des collections dans cet espace de noms. Vous trouverez plus d'informations sur les configurations d'ansible-galaxy <1>ici</1><2> </2><3/>."
+msgstr "<0>Remarque : Utilisez cette URL pour configurer ansible-galaxy afin de télécharger des collections dans cet espace de noms. Vous trouverez plus d'informations sur les configurations d'ansible-galaxy <1>ici</1><2> </2><3/>."
```
Comment (himdel) [inline locale/fr.po:68]:
```suggestion
msgstr "<0>Remarque:</0> Utilisez cette URL pour configurer ansible-galaxy afin de télécharger des collections dans cet espace de noms. Vous trouverez plus d'informations sur les configurations d'ansible-galaxy <1>ici</1><2> </2><3/>."
```

File: locale/fr.po [hunk 6]
```diff
@@ -468,42 +466,42 @@ msgstr "Code"
...
 
 #: src/containers/namespace-detail/namespace-detail.tsx:407
 msgid "Collection \"{0}\" has been successfully deprecated."
-msgstr ""
+msgstr "La collection  \"{0}\"  a été dépréciée."
 
 #: src/containers/namespace-detail/namespace-detail.tsx:406
 msgid "Collection \"{0}\" has been successfully undeprecated."
-msgstr ""
+msgstr "La collection  \"{0}\"  a été supprimée."
```
Comment (himdel) [inline locale/fr.po:485]: supprimée means removed, right? I think this is missing the info that we're not removing the collection, only the depreciation of it?

```suggestion
msgstr "La dépréciation de la collection  \"{0}\"  a été supprimée."
```

File: locale/fr.po [hunk 7]
```diff
@@ -1808,15 +1802,15 @@ msgstr "Les espaces de noms apparaîtront une fois créés"
...
 #: src/containers/certification-dashboard/certification-dashboard.tsx:209
 msgid "Needs Review"
-msgstr "Examen des besoins"
+msgstr "Examen Besoins"
 
-#: src/containers/certification-dashboard/certification-dashboard.tsx:344
+#: src/containers/certification-dashboard/certification-dashboard.tsx:340
 msgid "Needs review"
-msgstr ""
+msgstr "Examen des besoins"
```
Comment (himdel) [inline locale/fr.po:1809]: same issue as in the spanish version

```suggestion
msgstr "Besoin de révision"
```

(and the same right above)

File: locale/fr.po [hunk 8]
```diff
@@ -3441,42 +3422,42 @@ msgstr "{0} a été ajouté"
...
-#: src/components/headers/collection-header.tsx:449
+#: src/components/headers/collection-header.tsx:424
 #: src/components/headers/partner-header.tsx:46
 msgid "{company} logo"
 msgstr "{company} logo"
 
 #: src/components/numeric-label/numeric-label.tsx:84
 msgid "{count, plural, one {Dependency} other {Dependencies}}"
-msgstr ""
+msgstr "{count, plural, one {Dependency} autre {Dependencies}}"
```
Comment (himdel) [inline locale/fr.po:3439]: for plural translations, only the bits inside {} should be translated...

```suggestion
msgstr "{count, plural, one {Dépendance} other {Dépendances}}"
```

Also, the bits inside none of these are translated here, is that intentional? 

(5 in a row)

File: locale/ja.po [hunk 9]
```diff
@@ -3441,42 +3422,42 @@ msgstr "{0} が追加されました"
...
-#: src/components/headers/collection-header.tsx:449
+#: src/components/headers/collection-header.tsx:424
 #: src/components/headers/partner-header.tsx:46
 msgid "{company} logo"
 msgstr "{company} ロゴ"
 
 #: src/components/numeric-label/numeric-label.tsx:84
 msgid "{count, plural, one {Dependency} other {Dependencies}}"
-msgstr ""
+msgstr "{count, plural, one {依存関係} other {依存関係}}"
```
Comment (himdel) [inline locale/ja.po:3439]: (these are correct, the linter is mishandling these, I'll fix the linter)

(5 in a row)

File: locale/ko.po [hunk 10]
```diff
@@ -3311,25 +3292,25 @@ msgstr "예, 중지합니다"
 
 #: src/components/signing/sign-all-certificates-modal.tsx:40
 #~ msgid "You are about to sign <0>all</0> versions under <1>{name}</1>."
-#~ msgstr "<1>{name}</1>에서 <0>all</0> 버전에 서명하려고 합니다."
+#~ msgstr "You are about to sign <0>all</0> versions under <1>{name}</1>."
 
 #: src/components/signing/sign-single-certificate-modal.tsx:40
 msgid "You are about to sign <0>version {version}</0> under <1>{name}</1>."
-msgstr "<1>{name}</1>에서 <0>version {version}</0> 에 서명하려고 합니다."
+msgstr "<1>{name}</1>에서 <0>version {version}에 서명하려고 합니다."
```
Comment (himdel) [inline locale/ko.po:3299]:
```suggestion
msgstr "<1>{name}</1>에서 <0>version {version}</0> 에 서명하려고 합니다."
```

(and I suspect the first `version` should become something like `버전에`)

File: locale/ko.po [hunk 11]
```diff
@@ -3441,33 +3422,33 @@ msgstr "{0}이/가 추가되었습니다"
...
-#: src/components/headers/collection-header.tsx:449
+#: src/components/headers/collection-header.tsx:424
 #: src/components/headers/partner-header.tsx:46
 msgid "{company} logo"
 msgstr "{company} 로고"
 
 #: src/components/numeric-label/numeric-label.tsx:84
 msgid "{count, plural, one {Dependency} other {Dependencies}}"
-msgstr ""
+msgstr "{count, plural, one {Dependency} other {Dependencies}}"
```
Comment (himdel) [inline locale/ko.po:3439]: (Dependency, Module, Plugin and Role are untranslated here as well (and byte))

File: locale/nl.po [hunk 12]
```diff
@@ -701,11 +698,11 @@ msgstr "Een nieuwe namespace maken"
 
 #: src/components/user-form/user-form.tsx:97
 #~ msgid "Create a password using at least 9 characters, including special characters , ex <!@$%>. Avoid using common names or expressions."
-#~ msgstr "Maak een wachtwoord van ten minste 9 tekens, inclusief speciale tekens , bijvoorbeeld <!@$%>. Vermijd het gebruik van veelvoorkomende namen of uitdrukkingen."
+#~ msgstr "Create a password using at least 9 characters, including special characters , ex <!@$%>. Avoid using common names or expressions."
 
 #: src/components/user-form/user-form.tsx:99
 msgid "Create a password using at least {minLength} characters, including special characters , ex <!@$%>. Avoid using common names or expressions."
-msgstr ""
+msgstr "Maak een wachtwoord van ten minste 9 tekens, inclusief speciale tekens , bijvoorbeeld {minLength}. Vermijd het gebruik van veelvoorkomende namen of uitdrukkingen."
```
Comment (himdel) [inline locale/nl.po:705]:
```suggestion
msgstr "Maak een wachtwoord van ten minste {minLength} tekens, inclusief speciale tekens , bijvoorbeeld <!@$%>. Vermijd het gebruik van veelvoorkomende namen of uitdrukkingen."
```

File: locale/nl.po [hunk 13]
```diff
@@ -3317,21 +3297,21 @@ msgstr "Ja, stoppen"
 
 #: src/components/signing/sign-single-certificate-modal.tsx:40
 msgid "You are about to sign <0>version {version}</0> under <1>{name}</1>."
-msgstr "You are about to sign <0>version {version}</0> under <1>{name}</1>."
+msgstr "U staat op het punt om <0>versie {version}</0> te ondertekenen onder <1>{name}</1>."
 
 #: src/components/signing/sign-all-certificates-modal.tsx:58
 msgid "You are about to sign <0>{numberOfAffected}</0> version(s) under <1>{name}</1>."
-msgstr ""
+msgstr "U staat op het punt om <0>versie {numberOfAffected}</0> te ondertekenen onder <1>{name}</1>."
```
Comment (himdel) [inline locale/nl.po:3304]: the wrapping is likely wrong here, we want to "sign **N** versions", not "sign **version N**

perhaps something like..
```suggestion
msgstr "U staat op het punt om <0>{numberOfAffected}</0> versies te ondertekenen onder <1>{name}</1>."
```
?

File: locale/nl.po [hunk 14]
```diff
@@ -3443,42 +3423,42 @@ msgstr "{0} werd toegevoegd"
...
 
 #: src/components/numeric-label/numeric-label.tsx:76
 msgid "{count, plural, one {Role} other {Roles}}"
-msgstr ""
+msgstr "{count, plural, one {Role} other{Roles}}"
 
 #: src/utilities/get_human_size.ts:14
 msgid "{n, plural, one {# byte} other {# bytes}}"
-msgstr "{n, plural, one {# byte} other {# bytes}}"
+msgstr "{n, plural, one {# byte} other{# bytes}}"
```
Comment (himdel) [inline locale/nl.po:3456]: likely also syntax error without the space

```suggestion
msgstr "{n, plural, one {# byte} other {# bytes}}"
```

(and as with the other files (except ja), these are missing translations)

File: locale/es.po (hunk 1 above)
Comment (adityamulik) [reply locale/es.po]: language header has been fixed as per ba6745fd2bcd470adf7e405eb3ceffe07b4efc3c
Will check in the automation script to update and fix for modifying the headers as per the language.

File: locale/es.po [hunk 15]
```diff
@@ -1812,13 +1805,13 @@ msgstr "Los espacios de nombres aparecerán una vez creados"
 msgid "Needs Review"
 msgstr "Revisión de necesidades"
 
-#: src/containers/certification-dashboard/certification-dashboard.tsx:344
+#: src/containers/certification-dashboard/certification-dashboard.tsx:340
 msgid "Needs review"
-msgstr ""
+msgstr "Revisión de necesidades"
```
[the remark this answers is not in the document]
Comment (adityamulik) [reply locale/es.po]: I have removed the es.po and nl.po files as the translation wasn't completed.

File: locale/es.po (hunk 3 above)
Comment (adityamulik) [reply locale/es.po]: I have removed the es.po and nl.po files as the translation wasn't completed.

File: locale/nl.po (hunk 12 above)
Comment (adityamulik) [reply locale/nl.po]: I have removed the es.po and nl.po files as the translation wasn't completed.

File: locale/nl.po (hunk 13 above)
Comment (adityamulik) [reply locale/nl.po]: I have removed the es.po and nl.po files as the translation wasn't completed.

**Pull request closed without merging**